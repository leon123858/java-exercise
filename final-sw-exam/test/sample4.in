student 2001 2002 2003 2004 2005
schoolStrategy Excellent,3 Competent,2 NeedsWork,1 Trash,0
designCriterion A1 criteria2_A1.txt
designCriterion A1 criteria1_A1.txt
designCriterion A2 criteria2_A2.txt
assignment A1 2001 2002,score2_A1_1.txt 2003,score2_A1_1.txt 2004,score2_A1_2.txt 2005,score2_A1_3.txt
assignment A1 2001 2002,score2_A1_1.txt 2003,score2_A1_1.txt 2004,score2_A1_2.txt 2005,score2_A1_3.txt
assignment A1 2002 2001,score2_A1_2.txt 2003,score2_A1_2.txt 2004,score2_A1_2.txt 2005,score2_A1_1.txt
assignment A1 2003 2001,score2_A1_3.txt 2004,score2_A1_3.txt 2005,score2_A1_2.txt