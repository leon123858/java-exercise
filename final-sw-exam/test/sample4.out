Error
Error
A1-2001 was reviewed by 2002. Level: Excellent Competent Excellent Competent
A1-2001 was reviewed by 2003. Level: Excellent Competent Excellent Competent
A1-2001 was reviewed by 2004. Level: Competent NeedsWork Competent NeedsWork
A1-2001 was reviewed by 2005. Level: Trash Excellent Trash Trash
Error
A1-2002 was reviewed by 2001. Level: Competent NeedsWork Competent NeedsWork
A1-2002 was reviewed by 2003. Level: Competent NeedsWork Competent NeedsWork
A1-2002 was reviewed by 2004. Level: Competent NeedsWork Competent NeedsWork
A1-2002 was reviewed by 2005. Level: Excellent Competent Excellent Competent
A1-2003 was reviewed by 2001. Level: Trash Excellent Trash Trash
A1-2003 was reviewed by 2004. Level: Trash Excellent Trash Trash
A1-2003 was reviewed by 2005. Level: Competent NeedsWork Competent NeedsWork